v22.1/alter-range-relocate.md

```diff
@@ -0,0 +1,228 @@
+---
+title: ALTER RANGE ... RELOCATE
+summary: Use the ALTER RANGE ... RELOCATE statement to move a lease or replica between stores in an emergency situation.
+toc: true
+docs_area: reference.sql 
+---
+
+<span class="version-tag">New in v22.1:</span> The `ALTER RANGE ... RELOCATE` statement is a subcommand of [`ALTER RANGE`](alter-range.html). It is used to move a lease or [replica](architecture/overview.html#architecture-replica) between [stores](cockroach-start.html#store). This is helpful in an emergency situation to relocate data in the cluster.
```

Comment:
I wonder if we should link this to the similar ALTER TABLE EXPERIMENTAL_RELOCATE commands, which are a key based approach to moving replicas and leases around.